modules/system-info/reporters/server.php

```diff
@@ -410,4 +383,75 @@ public function get_elementor_library() {
 			'value' => 'Connected',
 		];
 	}
+
+	/**
+	 * Check write permissions by provided paths.
+	 *
+	 * @since 3.7.0
+	 * @access public
+	 *
+	 * @param array $paths_to_check
+	 * @param array $write_problems
+	 *
+	 * @return array {
+	 *      @type string value: Writing permissions status.
+	 *      @type bool   warning: Whether to display a warning. True if some required.
+	 * }
+	 */
+	public function check_write_permissions( array $paths_to_check, $write_problems = [] ) {
```

Comment:
try to do something like this:
```PHP
function get_permissions( $paths ) {
    // ...

    return [
        'path-1' => [ 
            'read' => true, 
            'write' => false, 
            'execute' => true,
         ],
    ];
}
```

and change the original code to use this and return its old data

Reply:
Done